shared-libs/cht-datasource/src/person.ts

```diff
@@ -32,14 +32,23 @@ export namespace v1 {
     }
   };
 
-  const assertTypeQualifier: (
+  const assertTypeQualifier: (qualifier: unknown) => asserts qualifier is ContactTypeQualifier = (
     qualifier: unknown
-  ) => asserts qualifier is ContactTypeQualifier = (qualifier: unknown) => {
+  ) => {
     if (!isContactTypeQualifier(qualifier)) {
       throw new Error(`Invalid type [${JSON.stringify(qualifier)}].`);
     }
   };
 
+  const assertLimitAndSkip = (limit: unknown, skip: unknown) => {
+    if (typeof limit !== 'number' || limit <= 0) {
+      throw new Error('limit must be a positive number');
```

Comment:
Minor, but lets also print out the invalid value.  Something like: 

```suggestion
      throw new Error(`The limit must be a positive integer: [${limit}].`);
```